api/script/index.js

```diff
@@ -33,7 +33,7 @@ const processReversal = (bus, log, $meta, transfer) => {
         return bus.importMethod(method, {timeout: 30000})(transfer, $meta)
             .catch(error => {
                 if (error.type === 'transfer.transferAlreadyReversed') {
-                    return transfer;
+                    return error;
```

Comment:
Returning error (instead of throwing) does not look right. 
Please consult with @Kristiyan-Todorov, who was working on the similar issue and needed the details from the transfer to be able to return proper ISO response. Check this change: 8588b979

Also let's not merge in master.

Reply:
Returning transfer does not look right too.
The return value should be the result of reversal/adjustment, not the transfer itself.

This will be inconsistent, because:
1. If there is no error, we will expect reversal result object value to be returned
2. If there is error of type 'transfer.transferAlreadyReversed', we will expect transfer object value to be returned

So i think it is better to return just empty object, but not transfer.

Also, look a few lines below https://github.com/softwaregroup-bg/ut-transfer/blob/8588b979deeda1cbf86600780900683e32ff99de/api/script/index.js#L43

Closing this PR. Will make patch branch.